pep/app/pods/components/page/sidebar/widgets/sign-in/template.hbs

```diff
@@ -0,0 +1,63 @@
+<div class="bg-primary border-divider-b" {{did-insert this.onElementInsert}}>
+    <div class="p-3">
+        <div class="d-flex justify-content-between align-items-center mb-3">
+            <h5 class="mb-0">
+                {{t "relatedInfo.widgets.signIn.title"}}
+            </h5>
+        </div>
+        <ContentWithPlaceholder @placeholderComponent="loading/widget" @isLoading={{this.loadResults.isRunning}}>
```

Comment:
@napafundi ideally we would create a component just for the login form so that all this logic is not duplicated

Reply:
@bakerac4 Yeah I agree. I think it might be better to wait to extract that into a component when this is done https://github.com/Psychoanalytic-Electronic-Publishing/PEP-Web-User-Interface/issues/311 but if you think it should be done first I can knock it out